foodsaving/groups/tasks.py

```diff
@@ -68,8 +69,8 @@ def send_summary_emails():
                     email.send()
                     email_count += 1
                     email_recipient_count += len(email.to)
-                except Exception:
-                    traceback.print_exc()
+                except AnymailAPIError:
+                    sentry_client.captureException()
```

Comment:
Unrelated change I snuck in, seems a better approach I borrowed from another place in the code.